src/components/TextArea/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+export const textAreaDataTestIds = {
+  base: 'text-area__base',
+};
+
+type TextAreaProps = {
+  onValueChange?: (value: string) => void;
+};
+
+const TextArea = ({ onValueChange }: TextAreaProps): JSX.Element => {
+  const handleOnChange = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+    event.preventDefault();
+
+    onValueChange?.(event.target.value);
```

Comment:
it's kinda odd to use the conditional chaining operator on executing function 🤔 
But can I know which condition the `onValueChange` is not passed?

Reply:
@nvminhtue I use the conditional chaining operator for null-safety because `onValueChange` can be null 😉 
![image](https://github.com/manh-t/react-survey/assets/60863885/fce41b5e-04bb-4dc1-a259-066b67f86d48)